lib/composers/moveToComposer.js

```diff
@@ -1,12 +1,43 @@
 const {makeModifierComposer} = require('../utils/makeComposer');
 const composers = require('../constants/composer');
+const {validate, validators} = require('../validation');
+const {invalidInputMessage} = require('../texts');
 
 /**
  * Defines moveTo method
  */
-const moveToComposer = makeModifierComposer(composers.MOVE_TO, ['moveTo'], (_, meta) => ({
-	...meta,
-	isMoveTo: true,
-}));
+const moveToComposer = makeModifierComposer(
+	composers.MOVE_TO,
+	['moveTo'],
+	/**
+	 *
+	 * @param {number} x
+	 * @param {number} y
+	 */
+	(_, meta, x, y) => {
+		if (x || y) {
```

Comment:
if `x` and `y` will be `empty string` or `null` or other falsy values - this condition will be skiped and user will not see any error. Pls fix it.

Reply:
This condition will not pass if _x_ and _y_ will be 0. Pls fix it